src/test/java/com/training/web/tests/rccl/RCCLTest.java

```diff
@@ -0,0 +1,100 @@
+package com.training.web.tests.rccl;
+
+import com.codeborne.selenide.Selenide;
+import com.google.gson.Gson;
+import com.training.base.BaseTest;
+import com.training.services.ga.authenticate.RequestBody;
+import com.training.services.ga.authenticate.Response;
+import com.training.utilities.RestEngine;
+import com.training.web.pages.rccl.SigInPage;
+import com.training.web.pages.rccl.UserAccountPage;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.training.utilities.UniqueMailId.getUniqueMailId;
+
+public class RCCLTest extends BaseTest {
+    private Map<String, String> headerMap;
+    private RequestBody requestBodyAuthenticate;
+    private Response authenticationResponse;
+    private com.training.services.ga.loyalty.RequestBody requestBody;
+    private String userPassword = "M212adasdasjdklj!";
+    private String userEmail = getUniqueMailId();
+    private String dateOfYear = "1962";
+    private String answer = "NewYork";
+
+    @Test
+    public void testUserLoyality(){
+       Selenide.open(map.get("rccl_url") + "/account/");
```

Comment:
instead of appending /account here, you can add this directly to json because that is our starting point for all tests